text/machine_comprehension/gpt-2/README.md

```diff
@@ -0,0 +1,76 @@
+# GPT-2
+
+## Use-cases
+Transformer-based language model for text generation.
+
+## Description
+[GPT-2](https://openai.com/blog/better-language-models/) is a large transformer-based language model with a simple objective: predict the next word, given all of the previous words within some text.
+
+## Model
+
+ |Model        |Download  |Checksum| Download (with sample test data)|ONNX version|Opset version|Accuracy |
+|-------------|:--------------|:--------------|:--------------|:--------------|:--------------|:--------------|
+|GPT-2       |[522.81 MB](https://onnxzoo.blob.core.windows.net/models/opset_10/GPT2/model.onnx) | [MD5](https://onnxzoo.blob.core.windows.net/models/opset_10/GPT2/gpt2-md5.txt)| [438.3 MB](https://onnxzoo.blob.core.windows.net/models/opset_10/GPT2/GPT2.tar.xz)| 1.6 | 10 |mAP of [0.024](https://docs.google.com/spreadsheets/d/1sryqufw2D0XlUH4sq3e9Wnxu5EAQkaohzrJbd5HdQ_w/edit#gid=0)|
+
+
+## Inference
+
+### Input to model
+Sequence of words as a string. Example: "Here is some text to encode : Hello World", tokenized by Byte-Pair-Encoding.
+**input_ids**: Indices of input tokens in the vocabulary.
+**past**: precomputed hidden-states
+Other optional inputs to the model are:

```

Comment:
Based on configurations the original model could have optional inputs and outputs.
On the other hand, in this page the uploaded ONNX model should have already chosen a specific configuration, and has fixed number of inputs/outputs. Let's note them down here.

Reply:
Sure. Makes sense.